frontend/src/app/shared/components/button/button.component.html

```diff
@@ -1,7 +1,9 @@
 <button 
     [ngStyle]="{
         height: height,
-        width: width
+        width: width,
+        fontSize: fontSize
     }"
+    class="{{btnClass}} base-button"
```

Comment:
I think we shouldn't pass btnClass directly, you can use some parameter and define a class based on it, for example:
`<app-button text="Sign In" width="250px" variant="outlined"></app-button>` 

Moreover, this parameter should have a default value. The width, height, and fontSize will be rarely passed, while more often, we'll only pass the text and the parameter if needed.